### PR TITLE
feat(multiselect): Added multiselect accessibility - OEL-2123

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

**Review by brummbar — CHANGES_REQUESTED** [review]
The script added works only on the story, right? It needs to be shipped and activated by users. It probably needs to be in a dedicated JS, so that in Drupal we can include it together with the library.

By the way, what about this option https://slimselectjs.com/settings#allowDeselect ?
It allows to deselect when hovering an entry.

File: src/components/bcl-select/select.story.js [hunk 1]
```diff
@@ -58,10 +58,36 @@ const initMultiselect = (story) => {
   return `
     <script>
       if (document.querySelector(".multi-select")) {
-        new SlimSelect({
+        function setAccesibilty(el, selectedVal) {
```
[the remark this answers is not in the document]
Comment (tibi2303) [reply src/components/bcl-select/select.story.js]: I've included your feedback in this new commit - create a new file multiselect.js that it's inside the package.
Also fixed the styles for icons, mobile, error/valid states and fixed a possible bug when multiple multiselect are inside the pages to init all. Please when you can have a look

[the remark this answers is not in the document]
Comment (tibi2303) [reply src/components/bcl-select/select.story.js]: Also this allowDeselect is for removing all items not each one individual.

**Review by drishu — CHANGES_REQUESTED** [review]
I dont see anything wrong with it per se, but when I integrated it into oe_showcase it made the form completely disappear.
Steps to repro:
- install oe_showcase OEL-1574
- create a project listing page
- override the slimselect.min.js
- override the oe-bcl-default.css
- (adding the multiselect.js is optional, is not relevant to the issue but you can also add it)
- reload the listing page, the form should be gone

**Pull request closed without merging**